### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by airenzp**
This PR updates the pedigree graph so that it uses elbow arrows without inheriting the color from the node and now the parents are on an upper level. It also adds some small fixes, including returning to a page after login and updating the type of the pedigree input parameter for the PedigreeNetwork, solving the issue #114.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,7 +44,12 @@ f62cebbc2e43        herdbook_main              "/bin/sh -c /entrypo…"   3 hour
 
 ```
 
-To access the local server deployed open this url https://localhost:8443 in your browser. You will need to configure the browser to allow self-signed localhost certificates. In Chrome, this can be done by accessing this property from the browser: `chrome://flags/#allow-insecure-localhost` and setting its vale to Enabled. In Firefox, when loading the url you can click on Advanced and  add the exception suggested from the browser. You can also use http://localhost:8080.
+To access the local server deployed open this url https://localhost:8443 in your browser. You will need to 
+configure the browser to allow self-signed localhost certificates. In Chrome, this can be done by accessing 
+this property from the browser: `chrome://flags/#allow-insecure-localhost` and setting its vale to Enabled. 
+In Firefox, when loading the url you can click on Advanced and  add the exception suggested from the browser. 
+You can also use http://localhost:8080 to access the frontend. Note though that if you edit frontend files at runtime, 
+recompilation changes are only visible at the dev port: 8443.
```
Comment (danr) [inline README.md]: Hmm I get live hot reload updates on 8080 as well as 8443, so I don't think this readme change is correct.

File: frontend/src/login.tsx [hunk 2]
```diff
@@ -29,7 +29,8 @@ export function Login() {
     login(username, password).then(
       status => {
         if (status == 'logged_in') {
-          history.push("/")
+          //history.push("/")
```
Comment (danr) [inline frontend/src/login.tsx]: Better remove this comment before Martin sees it! :sweat_smile:

File: frontend/src/individual_pedigree.tsx [hunk 3]
```diff
@@ -85,7 +84,8 @@ export function IndividualPedigree({ id, generations }: { id: string }) {
 
               </dl>
             </TableCell>
-            <TableCell width="85%" >
+            <TableCell width="85%" style={{textAlign: "left"}}>
+              <h3>Pedigree: <input value={generations_input} onChange={event => setGenerations(event.target.value)} type="number" min="1" max="50" /></h3>
               <PedigreeNetwork pedigree={pedigree} />
```
Comment (danr) [inline frontend/src/individual_pedigree.tsx]: Do we need to check that `pedigree !== undefined` first?

File: frontend/src/navigation_header.tsx [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+const useStyles = makeStyles({
+  root: {
+    flexGrow: 1,
+  },
+  main: {
+    height: "calc(100% - 72px)",
+  }
+});
+
+export function NavigationHeader() {
```
Comment (danr) [inline frontend/src/navigation_header.tsx]: There is no need for this DataContext-free Navigation anymore, right?

Comment (airenzp) [reply frontend/src/navigation_header.tsx]: It must have been added again after some merge. I removed it again

File: frontend/src/individual_pedigree.tsx (hunk 3 above)
Comment (airenzp) [reply frontend/src/individual_pedigree.tsx]: Jonas suggested to render the individual data first, that is why I removed the validation, I added it now to the pedigree
```
{pedigree && <PedigreeNetwork pedigree={pedigree} />}
```

File: frontend/src/login.tsx (hunk 2 above)
Comment (airenzp) [reply frontend/src/login.tsx]: Martin wanted the go back functionality, I do not understand :)

File: README.md (hunk 1 above)
Comment (airenzp) [reply README.md]: yes, it does now, wonder why the other day it did not

Comment (airenzp) [reply README.md]: I removed the comment in the README

File: frontend/src/login.tsx (hunk 2 above)
Comment (danr) [reply frontend/src/login.tsx]: Martin usually wants me to remove code in comments in my PRs. I don't mind too much though

Comment (norling) [reply frontend/src/login.tsx]: Yeah - it's not my strongest held opinion, but to me it just looks messy to have old commented code laying around. :)

**Review by norling — APPROVED** [review]
Looks nice!

**Comment by danr** [comment]
:tada:

**Pull request merged**